version.yaml

```diff
@@ -1,4 +1,4 @@
-current_version: 3.0.1.dev (Teal Blauwbok)
+current_version: 3.0.0.dev (Teal Blauwbok)
```

Comment:
@spirillen Hehe you are not in an isolated environment :wink:

Reply:
Well haven't noticed this little trick of your... I was in this case looking at your tag/3.0.0 commit for version number changes :package:

Reply:
@spirillen 
```
$ cd /path/to/PyFunceble
$ virtualenv -p python3 venv
$ source venv/bin/activate
$ pip3 install -e .
[... Edit PyFunceble ...]
$ PyFunceble --prod
[... Do what you have to do ...] :wink: `

Reply:
Well I'll write me a note on that one :+1:

Reply:
Have changed this back